### ISSUE TITLE
About Align image

### ISSUE THREAD

**Issue by szgy66** [minshu-kim/Neural-Image-Stitching#7]
Dear author, thank you for making your code public. I also got better results when I tested my own data with your code, thanks again. But the result I get is the result after stitching, may I ask how to operate if I want to obtain the conversion result from the reference image to the target image。Thanks!

**Comment by minshu-kim** [comment]
Thanks for your interest.
The warped image can be obtained by stitching a blank image (consisting of 0s) and a warped image.

For example, in our [stitch.py](https://github.com/minshu-kim/Neural-Image-Stitching/blob/main/stitch.py) code, 1) modify this [line](https://github.com/minshu-kim/Neural-Image-Stitching/blob/77a96f582ab96435c796e74419a89dba955acfd6/stitch.py#L76) as shown below to obtain the aligned images:
```
        fea_ref_w *= ref_mask.cpu()
        fea_tgt_w *= tgt_mask.cpu()
        fea = torch.cat([torch.zeros_like(fea_tgt_w), fea_tgt_w], dim=1).cuda()
```

and for masking valid regions, 2) revise this [line](https://github.com/minshu-kim/Neural-Image-Stitching/blob/77a96f582ab96435c796e74419a89dba955acfd6/stitch.py#L207) as below:
```
    pred = ((pred + 1)/2).clamp(0,1) * stit_mask.reshape(b, 1, *sizes) * tgt_mask
```

**Comment by wudami** [comment]
您好，请问您的问题解决了吗？如何得到warped图像，如果像作者所说的
`        fea_ref_w *= ref_mask.cpu()
        fea_tgt_w *= tgt_mask.cpu()
        fea = torch.cat([torch.zeros_like(fea_tgt_w), fea_tgt_w], dim=1).cuda()`
`    pred = ((pred + 1)/2).clamp(0,1) * stit_mask.reshape(b, 1, *sizes) * tgt_mask`
是否需要将seam_cut设为False?

**Comment by minshu-kim** [comment]
Hi @wudami,

You don't have to set it as False.
The intuition of the default setting as False is for observation of the network's blending. If you want to use NIS for practical purpose, I recommend you to use seam cutting.
